src/components/Calculator.js

```diff
@@ -4,8 +4,6 @@ import calculate from '../logic/calculate';
 function Calculator() {
```

Comment:
- [ ] [OPTIONAL] Great job so far @jmonto55 , However, I will recommend that you get familiar with using javascript arrow functions. It was introduced in ES6 and reduces lots of code and makes the code more readable.
Below is an example:
```
const Calculator = () => (
    // reference as a js object
      <div>
       // Calculator app design goes here
      </div>
);
```